### ISSUE TITLE
Lighthouse compose bug: error: 'mainnet' isn't a valid value for '--testnet <testnet>'

### ISSUE THREAD

**Issue by danbryan** [eth2-educators/eth2-docker#109]
When building lighthouse with

```
COMPOSE_FILE=lh-base.yml:oe.yml:lh-grafana.yml:shared-eth1.yml
```

I got the error of 
```
error: 'mainnet' isn't a valid value for '--testnet <testnet>'
[possible values: altona, medalla, spadina, zinken]
```

The current yaml has --testnet hard coded.
```
- --testnet
- ${NETWORK}
```

I don't think we want to remove it, because we want backwards compatibility with `Pyrmont`.  Maybe we could just do.
```
- ${USETESTNET}
```
where, if set, it would have a value like
```USETESTNET=--"testnet Pyrmont"```

**Comment by danbryan** [comment]
USETESTNET="--testnet Pyrmont" didn't work, Maybe a PR to request they allow testnet=mainnet would be the best fix.  Are most clients allowing the testnet flag?

**Comment by danbryan** [comment]
Fixed in LH 0.3.4
